### ISSUE TITLE
Final Report Peer Review - rjb392

### ISSUE THREAD

**Issue by TheButlah** [tinali0923/Volatility-Prediction#15]
What I liked:
- The dataset and problem was relevant and useful, and definitely was a big/messy data problem
- I liked how the team used PCA to clean up the data
- I liked the visualizations you used

What I think could use improvement:
- It felt like you just threw a bunch of linear models at the project to see what results you got. I would have liked to see some actual feature engineering happening, or at least a more detailed explanation of why some methods worked whereas others didn't.
- I would have liked to see more analysis of the particular features selected by the model. You did this for one feature at a time by training a model just on that feature, which is a good start. However, it would have been better to just do that analysis on the model that had all the features and then see what the relationships were between the various features and which ones were selected with the largest positive/negative weights.
- I find it hard to evaluate the results that you got based on just your loss function. First of all I think you are printing the total loss rather than mean loss (for example on the constant feature you get loss of 5 but that should be trivial for the network to reach near 0 loss on. Perhaps its near zero loss but for many entries so it sums to 5?). I believe that this is why your loss values are so high. Regardless, printing accuracy statistics too instead of just the loss values would be a more intuitive way to understand what your results were like.

Overall I think your project was good and I liked your techniques, but it felt a lot like you spent more effort throwing some models at a dataset rather than actually investigating the dataset.
